openmrs/api/src/main/java/org/projectbuendia/openmrs/api/db/ProjectBuendiaDAO.java

```diff
@@ -11,9 +11,14 @@
 
 package org.projectbuendia.openmrs.api.db;
 
+import org.openmrs.Encounter;
 import org.projectbuendia.openmrs.api.ProjectBuendiaService;
 
+import java.util.Date;
+import java.util.List;
+
 /** Database methods for {@link ProjectBuendiaService}. */
 public interface ProjectBuendiaDAO {
-    /* Add DAO methods here */
+    List<Encounter> getEncountersModifiedOnOrAfter(Date date);
```

Comment:
This is a tiny thing, but how about "at" instead of "on" (because "on" sustains the misunderstanding that Date is a date, but it isn't, it's a timestamp).